x-pack/test_serverless/functional/test_suites/common/management/data_views/_data_view_create_delete.ts

```diff
@@ -32,6 +37,22 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
         'x-pack/test_serverless/functional/es_archives/kibana_sample_data_flights_index_pattern'
       );
       await esArchiver.loadIfNeeded('test/functional/fixtures/es_archiver/logstash_functional');
+
```

Comment:
Matt, you could export the data view using the ui, and then save it under `test/functional/fixtures/kbn_archiver/?`, and then [load](https://github.com/elastic/kibana/blob/main/test/functional/apps/management/data_views/_scripted_fields_classic_table.ts#L51) / [unload](https://github.com/elastic/kibana/blob/main/test/functional/apps/management/data_views/_scripted_fields_classic_table.ts#L59).

Reply:
@wayneseymour I'm trying to avoid anything that relies on generic saved object importing and exporting. Seems like it should be possible, and long term we're trying to get rid of saved objects api.